services/frontend/src/components/ProjectCard/ProjectCard.js

```diff
@@ -1,12 +1,13 @@
 import React, {useMemo} from "react";
 import classNames from "classnames";
+import {Link} from "react-router-dom";
 import {Card, CardContent, IconButton} from "@material-ui/core";
 import {ChevronRight} from "@material-ui/icons";
 import "./ProjectCard.scss";
 
 const DISPLAY_SKILL_COUNT = 3;
 
-const ProjectCard = ({className, name, description, skills, isActive}) => {
+const ProjectCard = ({className, name, description, skills, isActive, id}) => {
```

Comment:
It'd be best if the `id` property was moved to just after `className` in the list.